man/dcast.data.table.Rd

```diff
@@ -95,20 +95,20 @@ dcast(DT, v1 + v2 + v3 ~ ., value.var = "v4")
 dcast(DT, v1 + v2 ~ ., value.var = "v4", fun.aggregate = sum)
 
 # fill and types
-dcast(DT, v2 ~ v3, value.var = 'v1', fill = 0L)  #  0L --> 0
-dcast(DT, v2 ~ v3, value.var = 'v4', fill = 1.1) # 1.1 --> 1L
+dcast(DT, v2 ~ v3, value.var = 'v1', fun.aggregate=length, fill = 0L)  #  0L --> 0
+dcast(DT, v2 ~ v3, value.var = 'v4', fun.aggregate=length, fill = 1.1) # 1.1 --> 1L
```

Comment:
and what about line width of manual examples? partial matching is nice here so we don't have to break down calls into multiple lines